static/js/components/PersonalTab.js

```diff
@@ -69,6 +78,9 @@ class PersonalTab extends React.Component {
         placeholder="Preferred language"
       /><br />
 
+      <Button raised onClick={this.saveAndContinue.bind(this)}>
+        Save and continue
+      </Button>
```

Comment:
I'm not sure `profileHasBeenSaved` feels right here since if they're successful, they're moved onto another tab.

Diffing `profile` and the edited copy is not difficult, it can be done with `_.isEqual`, but for simplicity's sake I'd like to only validate on button click for now